en/cpp/quickstart.md

```diff
@@ -85,33 +65,38 @@ git clone https://github.com/mavlink/MAVSDK.git --recursive
 cd MAVSDK
 ```
 
-Build the takeoff and land example:
-```sh
-cd examples/takeoff_land/
-mkdir build && cd build
-cmake ..
-make
+The examples can be found in the examples directory:
+```
+cd examples
 ```
 
+To e.g. build the takeoff and land example, you can do:
```

Comment:
"e.g." not needed here.

Reply:
Or more correctly "To build the takeoff and land example (for example), you can do:"